controllers/numaresourcesoperator_controller_test.go

```diff
@@ -59,6 +61,26 @@ const (
 	defaultOCPVersion = "v4.14"
 )
 
+var testPlatform platform.Platform
+
+func init() {
```

Comment:
Using init is problematic here, because it will trigger this function even on production code since it's part of the `controller` package.

Reply:
although this is initializing TEST_PLATFORM which is for tests use only, it is redundant for the manager code. I renamed the function and called it the main describe